### PR TITLE
Add descriptions for Remux

### PR DESCRIPTION
**Pull request by nyanmisaka**
based on https://github.com/jellyfin/jellyfin-web/pull/2069

**Changes**
* add descriptions for Remux.
* more translations for playback stats.
* little tweak for the transcode info in dashboard card.

### CODE REVIEW

**Review by Artiume — APPROVED** [review]

File: src/strings/en-us.json [hunk 1]
```diff
@@ -164,8 +164,8 @@
     "DetectingDevices": "Detecting devices",
     "DeviceAccessHelp": "This only applies to devices that can be uniquely identified and will not prevent browser access. Filtering user device access will prevent them from using new devices until they've been approved here.",
     "DirectPlaying": "Direct playing",
-    "DirectStreamHelp1": "The media is compatible with the device regarding resolution and media type (H.264, AC3, etc), but in an incompatible file container (mkv, avi, wmv, etc). The video will be re-packaged on the fly before being sent to the device.",
-    "DirectStreamHelp2": "Direct stream uses very little processing power with a minimal loss in video quality.",
+    "DirectStreamHelp1": "The media is in an incompatible file container (mkv, avi, wmv, etc) and the video stream is compatible with the device, but has an incompatible audio format (DTS, TRUEHD, etc) or audio channels (5.1, 7.1, etc). The video stream will be re-packaged losslessly on the fly before being sent to the device. Only audio stream will be transcoded.",
```
Comment (dkanada) [inline src/strings/en-us.json:167]:
```suggestion
    "DirectStreamHelp1": "The media is in an incompatible file container (MKV, AVI, WMV, etc) and the video stream is compatible with the device, but has an incompatible audio format (DTS, TRUEHD, etc) or number of audio channels. The video stream will be repackaged losslessly on the fly before being sent to the device. Only the audio stream will be transcoded.",
```

File: src/strings/en-us.json [hunk 2]
```diff
@@ -1430,5 +1430,16 @@
     "SubtitleVerticalPositionHelp": "Line number where text appears. Positive numbers indicate top down. Negative numbers indicate bottom up.",
     "Preview": "Preview",
     "LabelMaxMuxingQueueSize": "Max muxing queue size:",
-    "LabelMaxMuxingQueueSizeHelp": "Maximum number of packets that can be buffered while waiting for all streams to initialize. Try to increase it if you still encounter \"Too many packets buffered for output stream\" error in ffmpeg logs. The recommended value is 2048."
+    "LabelMaxMuxingQueueSizeHelp": "Maximum number of packets that can be buffered while waiting for all streams to initialize. Try to increase it if you still encounter \"Too many packets buffered for output stream\" error in ffmpeg logs. The recommended value is 2048.",
+    "Remuxing": "Remuxing",
+    "RemuxHelp1": "The media is in an incompatible file container (mkv, avi, wmv, etc). But both video stream and audio stream are compatible with the device. The media will be re-packaged losslessly on the fly before being sent to the device.",
```
Comment (dkanada) [inline src/strings/en-us.json:1435]:
```suggestion
    "RemuxHelp1": "The media is in an incompatible file container (MKV, AVI, WMV, etc) but both the video stream and audio stream are compatible with the device. The media will be repackaged losslessly on the fly before being sent to the device.",
```

**Comment by dkanada** [comment]
Does this replace that pull request or do we need both?

File: src/strings/en-us.json (hunk 1 above)
Comment (Artiume) [reply src/strings/en-us.json:167]: The file container could be compatible, it's all about the audio being incompatible.

**Comment by Artiume** [comment]
Both, his PR doesn't touch the file that mine touches.

**Review by MrTimscampi — APPROVED** [review]

**Review by Camc314 — APPROVED** [review]

**Pull request merged**